recipes/cad1/README.md

```diff
@@ -0,0 +1,91 @@
+# The 1st Cadenza Challenge
+Cadenza challenge code for the 1st challenge.
+
+For more information about the Cadenza Challenge please visit https://cadenzachallenge.com
```

Comment:
URL doesn't resolve, as its `.org`, we should also use Markdown links as GitHub renders Markdown files automatically.

```suggestion
For more information about the Cadenza Challenge please visit the [homepage](https://cadenzachallenge.org)
```

Reply:
I changed this README to contain only the baselines results.